renovate-config.json

```diff
@@ -48,6 +48,10 @@
         "https://github.com/ckeditor/ckeditor5/"
       ],
       "groupName": "ckeditor5"
+    },
+    {
+      "matchPackageNames": ["xmlsec"],
+      "allowedVersions": "<=1.3.13"
```

Comment:
Since it looks like the problem is specifically with `1.3.14` I would argue that rather than setting the specifer to `<=1.3.13` it should just be `!/1\.3\.14/` so that we can continue to get updates for newer releases, but avoid this specific problematic version.

Reply:
https://docs.renovatebot.com/configuration-options/#ignore-versions-with-negated-regex-syntax